README.md

```diff
@@ -6,37 +6,123 @@
+
+_The extension allows to zoom, rotate and flip the preview of the image. Inverting the colors of the preview is also supported._
+
+
+## Adding DICOM Viewer to Your oCIS Installation
+As oCIS administrator, you can add custom web applications for your users. By adding the DICOM Viewer to the oCIS WebUI, you enable your users to take advantage of the [functionalities of this web extension](#functionalities-of-dicom-viewer-web-extension).
+
+
+### Adding DICOM Viewer to the oCIS Deployment Example
+oCIS provides some [deployment examples](https://github.com/owncloud/ocis/tree/master/deployments/examples/) including detailed configuration step by step guides for [local production setup](https://doc.owncloud.com/ocis/next/depl-examples/ubuntu-compose/ubuntu-compose-prod.html) and [deployment of Infinite Scale on the Hetzner Cloud](https://doc.owncloud.com/ocis/next/depl-examples/ubuntu-compose/ubuntu-compose-hetzner.html).
+In both cases, it only takes three very small and simple steps to add the DICOM Viewer Web Extension to the [`ocis_full` deployment example](https://github.com/owncloud/ocis/tree/master/deployments/examples/ocis_full/) of your own installation:
+
+1. Navigate to the `/opt/compose/ocis/ocis_full` folder of your installation and copy [`dicom-viewer.yml`](https://github.com/owncloud/web-app-dicom-viewer/blob/main/dicom-viewer.yml) into the [`web_extensions`](https://github.com/owncloud/ocis/tree/master/deployments/examples/ocis_full/web_extensions) subfolder.
+
+2. Add `DICOMVIEWER=:web_extensions/dicom-viewer.yml` to the `## oCIS Web Extensions ##` section of the `.env` file of your installation (file is located in `/opt/compose/ocis/ocis_full`).
+
+Your `.env` file should now look like this:
+
+```
+## oCIS Web Extensions ##
+# It is possible to use the oCIS Web Extensions to add custom functionality to the oCIS frontend.
+# For more details see https://github.com/owncloud/web-extensions/blob/main/README.md
+
+<list of all web extensions>
+
+DICOMVIEWER=:web_extensions/dicom-viewer.yml
+
+```
+
+3. Append `${DICOMVIEWER:-}` to the `COMPOSE_FILE` variable at the very end of the last line of the `.env` file. This variable combines the configs of all the components that need to be loaded.
+
+`COMPOSE_FILE=docker-compose.yml${OCIS:-} ... <variables of lots of other configs that are added to docker compose> ... ${DICOMVIEWER:-}`
+
+After appending `${DICOMVIEWER:-}`, your `.env` file should look like this:
+
+```
+## IMPORTANT ##
+# This MUST be the last line as it assembles the supplemental compose files to be used.
+# ALL supplemental configs must be added here, whether commented or not.
+# Each var must either be empty or contain :path/file.yml
+COMPOSE_FILE=docker-compose.yml${OCIS:-}${TIKA:-}${S3NG:-}${S3NG_MINIO:-}${COLLABORA:-}${MONITORING:-}${IMPORTER:-}${CLAMAV:-}${ONLYOFFICE:-}${INBUCKET:-}${EXTENSIONS:-}${UNZIP:-}${DRAWIO:-}${JSONVIEWER:-}${PROGRESSBARS:-}${EXTERNALSITES:-}${DICOMVIEWER:-}
 
 ```
```

Comment:
````suggestion
3. Append `${DICOMVIEWER:-}` to the `COMPOSE_FILE` variable at the very end of the last line of the `.env` file. This variable combines the configs of all the components that need to be loaded.

   `COMPOSE_FILE=docker-compose.yml${OCIS:-} ... <variables of lots of other configs that are added to docker compose> ... ${DICOMVIEWER:-}`

   After appending `${DICOMVIEWER:-}`, your `.env` file should look like this:
   ```
   ## IMPORTANT ##
   # This MUST be the last line as it assembles the supplemental compose files to be used.
   # ALL supplemental configs must be added here, whether commented or not.
   # Each var must either be empty or contain :path/file.yml
   COMPOSE_FILE=docker-compose.yml${OCIS:-}${TIKA:-}${S3NG:-}${S3NG_MINIO:-}${COLLABORA:-}${MONITORING:-}${IMPORTER:-}${CLAMAV:-}${ONLYOFFICE:-}${INBUCKET:-}${EXTENSIONS:-}${UNZIP:-}${DRAWIO:-}${JSONVIEWER:-}${PROGRESSBARS:-}${EXTERNALSITES:-}${DICOMVIEWER:-}
   ```
````
This will make it render inline with the list item